src/pages/posts/use-github-actions-for-ci.md

```diff
@@ -0,0 +1,227 @@
+
+That leaves us with the following information: we're using `node@10`, `yarn`, and running the `test-ci-partial` script after installing our dependencies. For these actions, we'll use [nuxt/actions-yarn](https://github.com/nuxt/actions-yarn) which handily supports [different versions of Node.js](https://github.com/nuxt/actions-yarn#node-versions). Here's what that might look like:
+
+```hcl
+workflow "Test my code" {
+  on = "push"
+  resolves = ["test-ci-partial"]
+}
+
+action "Install dependencies" {
+  uses = "nuxt/actions-yarn@node-10"
+  args = "--frozen-lockfile"
+}
+
+action "test-ci-partial" {
+  needs = "Install dependencies"
+  uses = "nuxt/actions-yarn@node-10"
+  args = "run test-ci-partial"
+}
+```
+
+That should do it! By using an external action, we can keep our workflow nice and clean :nail_care: One thing to note is that `nuxt/actions-yarn` uses the full `FROM node` image - for the sake of optimization, you might consider forking the action and using a smaller base image.
+
+Here's a similar exercise with proven results in [JasonEtco/create-an-issue](https://github.com/JasonEtco/create-an-issue) of [replacing a basic `.travis.yml` file with a workflow](https://github.com/JasonEtco/create-an-issue/compare/d10d7bc2a567fa4288ead6b91f307aa4b44fb9f7...3b32e1e16d13ce431cc2ad4031eda7ba1396096a). Performance is important for CI, we want our tests to run quickly - so let's look at the difference in execution time:
+
+| Provider       | Execution time |
+|----------------|----------------|
+| Travis CI      | 36 seconds     |
+| GitHub Actions | 34 seconds     |
+
+They're basically the same! We haven't taken into account any of the speed improvements available to us in TravisCI, this is just the default behavior. Comparatively, I think that execution speed will only improve with GitHub Actions, and I'm really curious to see what kind of improvements users will get without any additional effort.
+
+## README Badges
+
+A beloved feature of most CI providers is their ability to show a badge on a repository's README, depicting the status of the build (whether its passing, failing, etc). I'm so used to the badges that if they aren't present my eyes get confused. Unfortunately, there's no first-class badge support with actions. I built [JasonEtco/action-badges](https://github.com/JasonEtco/action-badges) for this purpose; it works by [querying for the repository's Check Suites](https://developer.github.com/v3/checks/suites/#list-check-suites-for-a-specific-ref) and deriving a status by looking for the GitHub Action app's activity.
+
+```markdown
+![Build Status](https://action-badges.now.sh/JasonEtco/example)
+```
+
+## Where actions isn't perfect
+
+This post isn't intended to somehow prove that independant CI tools are made redundant by actions - just that for _some_ use-cases, you can choose between the two.
```

Comment:
```suggestion
This post isn't intended to somehow prove that independant CI tools are made redundant by Actions - just that for _some_ use-cases, you can choose between the two.
```

Reply:
OKAY WAIT @mscoutermarsh you'll actually know - is it always `Actions` not `actions`? Because it's `pull requests` not `Pull Requests`.

Reply:
There's mixed casing [here in our docs](https://developer.github.com/actions/#discovering-public-github-actions) and I don't know what to do 😭

Reply:
I reread this one a few times. 🤔 
I think it would depend on whether you're talking about Actions (proper noun) the platform vs actions (common noun) as units.

Reply:
I use Actions in all our copy.

But whatever, this is great, shippp 🚀

Reply:
WELL NOW I'M GONNA MAKE IT UPPERCASE EVERYWHERE but thanks for the 🚀 y'alls help is much appreciated